SimG4Core/PhysicsLists/src/CMSEmStandardPhysicsTrackingManager.cc

```diff
@@ -0,0 +1,731 @@
+#include "G4Version.hh"
```

Comment:
This is redundant, given that it is already included in the header file

Reply:
Not quite, then I'd have to move the include of `SimG4Core/PhysicsLists/interface/CMSEmStandardPhysicsTrackingManager.h` before the version check. I can do that if you want, but I think it's generally recommended to include-what-you-use in every implementation file...